monai/visualize/occlusion_sensitivity.py

```diff
@@ -18,7 +18,7 @@
 import torch.nn as nn
 
 from monai.networks.utils import eval_mode
-from monai.visualize import default_normalizer, default_upsampler
+from monai.visualize import default_upsampler
```

Comment:
```suggestion
from monai.visualize.visualizer import default_upsampler
```

Reply:
With that modification: `ModuleNotFoundError: No module named 'monai.visualize.visualizer.visualizer'; 'monai.visualize.visualizer' is not a package`

Reply:
I checked out this branch and this seems to work fine from my side: 
```py
# from monai.visualize import default_upsampler
from monai.visualize.visualizer import default_upsampler
```

Reply:
Apologies, you're right!